File: Modul-4/Soal-1/PRAK401-2410817320001-NazlaSalsabila.py
def main():
    kelipatan = int(input("Masukkan bilangan kelipatan yang dirubah menjadi simbol: "))
    simbol = input("Masukkan simbol yang akan menggantikan bilangan kelipatan: ")
    
    for i in range(1, 51):
        if i % kelipatan == 0:
            print(simbol, end=" ")
        else:
            print(i, end=" ")
    
main()

File: Modul-4/Soal-1/PRAK401-2410817320001-NazlaSalsabila.c
#include <stdio.h>

int main() {
    int kelipatan, i;
    char simbol[10];
    printf("Masukkan bilangan kelipatan yang dirubah menjadi simbol: ");
    scanf("%d", &kelipatan);
    printf("Masukkan simbol yang akan menggantikan bilangan kelipatan: ");
    scanf("%s", simbol);

    for (i = 1; i <= 50; i++) {
        if (i % kelipatan == 0) {
            printf("%s ", simbol);
        } else {
            printf("%d ", i);
        }
    }
    return 0;
}